src/test/resources/features/Kickouts.feature

```diff
@@ -275,4 +275,47 @@ Feature: Not Eligible for One Stop Shop
 #    Then the user is on the account-restore-error page
 
 
+  Scenario: User vat number have been excluded
```

Comment:
can we put fixed establishment in the scenario names for this and the other new scenario?

Reply:
Added fixed establishment to the name